File: src/test/java/com/eumji/zblog/ApplicationTest.java
package com.eumji.zblog;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

import static org.junit.Assert.*;
@RunWith(SpringRunner.class)
@SpringBootTest
public class ApplicationTest {
    @Value("${spring.datasource.password}")
    String url;
    @Test
    public void main() throws Exception {
        System.out.println(url);
    }

}

File: src/test/java/com/jchanghong/ApplicationTest.kt
package com.jchanghong

import org.junit.Test
import org.junit.runner.RunWith
import org.springframework.beans.factory.annotation.Value
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.junit4.SpringRunner

import org.junit.Assert.*

@RunWith(SpringRunner::class)
@SpringBootTest
class ApplicationTest {
    @Value("\${spring.datasource.password}")
    internal var url: String? = null

    @Test
    @Throws(Exception::class)
    fun main() {
      assertEquals(url,"0000")
    }

}